salt/states/pkg.py

```diff
@@ -130,6 +131,7 @@
         _namespaced_function(_get_latest_pkg_version, globals())
     _reverse_cmp_pkg_versions = \
         _namespaced_function(_reverse_cmp_pkg_versions, globals())
+    cmp_to_key = _namespaced_function(cmp_to_key, globals())
```

Comment:
Not sure you need this line.  Just adding `from functools import cmp_to_key `worked for me.